stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/header/viewmodel/MessageListHeaderViewModel.kt

```diff
@@ -67,7 +67,13 @@ public class MessageListHeaderViewModel(
      */
     public val channel: LiveData<Channel> =
         channelState.flatMapLatest { state ->
-            state.channelData.combine(state.members) { _, _ -> state.toChannel() }
+            combine(
```

Comment:
Reacting to watcher count change here to display the number of online users.

Reply:
That's a great catch, this was a bug previously.